experiment/update-hook/main.go

```diff
@@ -0,0 +1,39 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/cmd/hmac/updater"
+)
+
+func main() {
```

Comment:
It seems all other components are started with `logrusutil.ComponentInit()` and it seems to be setting logs to JSON format, so I added it instead of `logrus.SetFormatter(&logrus.JSONFormatter{})`

Reply:
I have tested running the `update-hook` command with `bazel run` and it works properly. I'll leave this `ComponentInit` unchanged.

Reply:
It looks like you might have accidentally removed it?

Reply:
Sorry my fault. Added it back.